web-app/src/main/webapp/script.js

```diff
@@ -55,3 +55,11 @@ function handleLocationError(browserHasGeolocation, infoWindow, pos) {
   );
   infoWindow.open(map);
 }
+
+window.onload = function() {
+  document.getElementById("report-form").style.visibility = "hidden";
+};
+
+function showReportForm() {
+  document.getElementById("report-form").style.visibility = "visible";
```

Comment:
We need to hide the home screen elements too! Also not sure if this is just outdated code, but screens are now denoted by two classes, home and report.

Reply:
Will do in later PR on seperate branch with updated UI!

Reply:
note that `visibility` doesn't hide the space that the elements occupy. Setting `display: none` will completely hide the element and also the space that it occupies.